zgw_consumers/models/validators.py

```diff
@@ -0,0 +1,58 @@
+from django.core.exceptions import ValidationError
+from django.core.validators import URLValidator
+from django.utils.deconstruct import deconstructible
+from django.utils.translation import gettext_lazy as _
+
+
+@deconstructible
+class StartWithValidator:
+    code = "invalid"
+
+    def __init__(
+        self,
+        prefix: str,
+        message: str = None,
+        code: str = None,
+        return_value: bool = True,
```

Comment:
I was initially a bit confused as to what `return_value` meant here, maybe we should rename the validator to `PrefixValidator` and replace `return_value` with something like `startswith`?